app/jobs/waste_notification_job.rb

```diff
@@ -13,7 +13,7 @@ def self.perform(waste_registration_id, waste_pickup_time_id)
 
     # Message to send
     title = "Abfallkalender"
-    parsed_date = Date.parse(waste_pickup_time.pickup_date).strftime("%d.%m.%Y")
+    parsed_date = waste_pickup_time.pickup_date.try(:strftime, "%d.%m.%Y")
```

Comment:
noch ein `return if parsed_date.blank?`

Reply:
ne ich glaube die push sollte trotzdem rausgehen, oder? es dürfte auch nicht auftreten, dass eine push anhand eines dates getriggert wird und dann hier das date nicht vorhanden ist

Reply:
Stimmt, die würde üben in Zeile 8 ja bereits rausfliegen